lighthouse-core/test/report/html/renderer/psi-test.js

```diff
@@ -20,10 +20,16 @@ const CriticalRequestChainRenderer =
     require('../../../../report/html/renderer/crc-details-renderer.js');
 
 const sampleResultsStr = fs.readFileSync(__dirname + '/../../../results/sample_v2.json', 'utf-8');
-const sampleResultsRoundtripStr = fs.readFileSync(
-  __dirname + '/../../../../../proto/sample_v2_round_trip.json',
-  'utf-8'
-);
+
+let sampleResultsRoundtripStr;
+try {
+  sampleResultsRoundtripStr =
+    fs.readFileSync(__dirname + '/../../../results/sample_v2.json', 'utf-8');
+} catch (err) {
+  // eslint-disable-next-line no-console
+  console.warn('Skipping test - you need to run yarn test-proto first.');
+}
+const itIf = sampleResultsRoundtripStr ? it : it.skip;
```

Comment:
this is a clever solution, though I'm a little worried it's too clever and we'll lose track that it's happening :)

I can only think of worse alternatives, though. Any `testPathIgnorePatterns` or clever globbing is much uglier and non-local to the actual code that depends on the round trip file existing.

What do you think about just a more explicit name, like `const itIfProtoExists` or something?